### PR TITLE
New Port Words With Freds

### PR DESCRIPTION
**Pull request by Fraxinus88**
# New Port for Words with Freds

## Game Information
- **Title**: Words with Freds
- **URL**: https://thatsmytrunks.itch.io/words-with-freds

### CFW Tests
Ensure your game has been tested on all major CFWs:
CFW Tests:
[X] AmberELEC
[X] ArkOS
[X] JELOS/ROCKNIX
[X] MuOS

Architecture
[X] armHF
[X] Panfrost (Mainline)

Resolution Tests
Test all major resolutions:
[X] 480x320
[X] 640x480
[X] 720x720 (RGB30)
[X] Higher resolutions (e.g., 1280x720

### CODE REVIEW

**Review by kloptops — DISMISSED** [review]
👍

File: ports/wordswithfreds/Words With Freds.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
```
Comment (Cebion) [inline ports/wordswithfreds/Words With Freds.sh:2]: Old header, please use the new header.
https://portmaster.games/packaging.html

File: ports/wordswithfreds/Words With Freds.sh [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+source $controlfolder/device_info.txt
+export PORT_32BIT="Y"
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+get_controls
+
+$ESUDO chmod 666 /dev/tty0
+
+GAMEDIR=/$directory/ports/wordswithfreds
+
+exec > >(tee "$GAMEDIR/log.txt") 2>&1
```
Comment (Cebion) [inline ports/wordswithfreds/Words With Freds.sh:21]: Try to use the new logging which overwrites itself 

```> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1```

**Review by Cebion — CHANGES_REQUESTED** [review]

File: ports/wordswithfreds/Words With Freds.sh (hunk 1 above)
Comment (Fraxinus88) [reply ports/wordswithfreds/Words With Freds.sh:2]: Okies, Updated, please review

File: ports/wordswithfreds/Words With Freds.sh (hunk 2 above)
Comment (Fraxinus88) [reply ports/wordswithfreds/Words With Freds.sh]: Okies, Updated, please review

**Review by Cebion — APPROVED** [review]

**Pull request merged**